mojave/items/misc/radios.dm

```diff
@@ -64,6 +64,8 @@
 	..()
 	weapon.play_tool_sound(src)
 	if(do_after(user, 30 SECONDS, target = src, interaction_key = DOAFTER_SOURCE_DECON))
+		var/drop_location = user.drop_location()
```

Comment:
This is for ham radios, not handhelds. You can't disassemble handhelds. This isn't really needed and will be kind of wonky.

Reply:
so true